enhancements/etcd/protecting-etcd-quorum-during-control-plane-scaling.md

```diff
@@ -0,0 +1,450 @@
+---
+title: protecting-etcd-quorum-during-control-plane-scaling
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@hexfusion"
+  - "@hasbro17"
+approvers:
+  - "@hexfusion"
+creation-date: 2021-08-20
+last-updated: 2021-11-18
+status: implementable
+see-also:
+  - "[Machine Deletion Hooks](https://github.com/openshift/enhancements/pull/862)"
+---
+
+# Protecting etcd Quorum During Control Plane Scaling
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+To enable automation of Control Plane scaling activities, in particular vertical scaling of the Control Plane Machines,
+we must implement a mechanism that protects etcd quorum and ensures the smoothest possible transition as new etcd
+members are added and old members removed from the etcd cluster.
+
+## Motivation
+
+As Red Hat expands its managed services offerings, the ability to safely increase and decrease the capacity of an
```

Comment:
the motivation sounds very much like scaling up to 5 masters and more is the goal. It is not (I hope), but this enhancement is about safely replacing nodes with bigger machines.

Reply:
Fair point, I'll clarify to make this more obvious we mean via vertical scaling of the control plane